data-prepper-plugins/http-source/build.gradle

```diff
@@ -5,7 +5,9 @@ plugins {
 dependencies {
     implementation project(':data-prepper-api')
     implementation project(':data-prepper-plugins:blocking-buffer')
+    implementation project(':data-prepper-plugins:common')
     implementation "com.linecorp.armeria:armeria:1.9.2"
+    implementation "commons-io:commons-io:2.11.0"
```

Comment:
We ought to move the versions of dependencies we use in multiple places out to the external file with the others that we've already defined.

Reply:
Right now we have a versionsMap in our `build-resources.gradle` to provide this consistency.

Gradle 7 should help us improve this, but it is not possible to update presently due to the OpenSearch plugin.

Reply:
We could add commons-io version into versionMap. There might be other common packages. We should do that in a separate PR.

Reply:
Please make a separate PR for this.